std/array.d

```diff
@@ -2016,11 +2019,40 @@ if (isDynamicArray!(E[]) && isForwardRange!R1 && isForwardRange!R2
     return app.data;
 }
 
+/// ditto
+E[] replace(E, R1, R2)(E[] subject, R1 from, R2 to, ref size_t countChanges)
+if (isDynamicArray!(E[]) && isForwardRange!R1 && isForwardRange!R2
+        && (hasLength!R2 || isSomeString!R2))
+{
+    import std.algorithm.searching : find;
+
+    countChanges = 0;
+    if (from.empty) return subject;
+
+    auto balance = find(subject, from.save);
+    if (balance.empty)
+        return subject;
```

Comment:
Not covered, Please install the CodeCov browser extension: https://github.com/codecov/browser-extension